### PR TITLE
DDF-1731: Update platform-configuration-listener for windows

### PR DESCRIPTION
**Pull request by garrettfreibott**
@codice/yellowteam

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/406" target="_blank">ReviewNinja

File: platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java [hunk 1]
```diff
@@ -125,7 +125,7 @@
     @Mock
     private Path configPath2;
 
-    private Path configFilePath = Paths.get(CONFIG_FILE_PATH);
+    private Path configFilePath = CONFIG_FILE_PATH;
```
Comment (andrewkfiedler) [inline platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java]: Looks like you could just remove this variable.

**Comment by andrewkfiedler** [comment]
:+1: LGTM

**Comment by andrewkfiedler** [comment]
selfie-0

selfie-1

**Comment by oconnormi** [comment]
:+1:

File: platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java (hunk 1 above)
Comment (garrettfreibott) [inline platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java]: Done

**Comment by garrettfreibott** [comment]
Passed, came up successfully, and processed valid .config files from the etc directory. :+1:

**Pull request merged**